ext/crypto/00_crypto.js

```diff
@@ -264,6 +264,18 @@
     return normalizedAlgorithm;
   }
 
+  function copyBuffer(input) {
```

Comment:
Thanks for deduplicating :-)

Reply:
I have merged this in https://github.com/denoland/deno/pull/13030 already to reduce the size of this PR.